src/testing/question.service.stub.ts

```diff
@@ -0,0 +1,10 @@
+import { of } from 'rxjs';
+import { QuestionQueueDto } from '../app/models/question-queue-dto';
+import { QuestionService } from '../app/services/question.service';
+
+export const QuestionServiceStub: Partial<QuestionService> = {
+
+    getQuestionFromQueue: jasmine.createSpy('getQuestionFromQueue')
```

Comment:
Nice. For reference, I figured out how to make modifiable stubs in unit tests. These are slightly better in that they allow you to configure the behaviour of the stub for the particular unit test in question. This can be very useful for testing how different kinds of errors are handled. Let me know if you want to implement this approach instead. I will certainly be moving towards that for the customer-facing ui